colab/widgets/templatetags/widgets_tag.py

```diff
@@ -9,5 +10,10 @@
 def import_widgets(context, area_id, widget_var=None):
     if not widget_var:
         widget_var = "widgets_{}".format(area_id)
+
     context[widget_var] = WidgetManager.get_widgets(area_id, context=context)
+
+    for widget in context[widget_var]:
+        widget.content = Template(widget.content).render(context)
```

Comment:
It seems that this render is doubled, you guys do the same operation here https://github.com/colab/colab/blob/improve_widget/colab/widgets/widget_manager.py#L37